src/shared/tpm2-util.c

```diff
@@ -3931,6 +3932,7 @@ static int tpm2_deserialize(
 }
 
 int tpm2_seal(Tpm2Context *c,
+              uint32_t handle_index,
```

Comment:
so here you call it the "handle_index", and earlier you called it the "handle". Please stick to one. (And as mentioned, I think we should rather call this storage_key or sealing_key or so)

Reply:
ack, changed it to `seal_key_handle` here as well.